mlos_bench/mlos_bench/config/schemas/environments/mock-env-subschema.json

```diff
@@ -19,12 +19,12 @@
                 },
                 {
                     "properties": {
-                        "seed": {
+                        "mock_env_seed": {
```

Comment:
Not sure I follow the logic of changing this particular one.  Why not let that draw from other global `seed` environment variables as well?

Reply:
Also, the `MockEnv` changes might be better in a different PR

Reply:
> Why not let that draw from other global seed environment variables as well?

Our `seed` parameter is also used to switch on the deterministic behavior. I want to separate the determinism in the optimizer and mock environment. Also, global parameters with short names like `seed` or `metrics` are prone to name collisions and our parameter substitution behavior makes it hard to debug. So yeah, naming things is hard 😺